File: led.c
#include <wiringPi.h>

const int LEDS[] = {15, 16, 17, 6, 19, 20, 21, 22};

int main()
{
    wiringPiSetupGpio();
    
    for(int i=0 ; i<sizeof(LEDS)/sizeof(LEDS[0]) ; ++i) {
        pinMode(LEDS[i], OUTPUT);    
    }

    // for(int i=0 ; i<8 ; ++i) {
    //     digitalWrite(LEDS[i], HIGH);        // HIGH = 1
    //     delay(100);
    //     digitalWrite(LEDS[i], LOW);
    //     delay(50);
    // }

    // for(int i=0 ; i<sizeof(LEDS)/sizeof(LEDS[0]) ; ++i) {
    for(int i=0 ; i<8 ; ++i) {
        digitalWrite(LEDS[i], HIGH);        // HIGH = 1
    }
    delay(2000);

    for(int i=0 ; i<8 ; ++i) {  
        digitalWrite(LEDS[i], LOW);
    }

    return 0;
}

File: led.py
from wiringpi import wiringPiSetupGpio, digitalWrite, pinMode, delay
from wiringpi import OUTPUT, HIGH, LOW

LEDS = [15, 16, 17, 6, 19, 20, 21, 22]


def main():
    wiringPiSetupGpio()
    
    for i in LEDS:
        pinMode(i, OUTPUT)

    for i in LEDS:
        digitalWrite(i, HIGH)        # HIGH = 1
    
    delay(2000)

    for i in LEDS: 
        digitalWrite(i, LOW)


if __name__ == "__main__":
    main()
